src/main/resources/templates/courses/assignments/group-pulls.ftl

```diff
@@ -0,0 +1,81 @@
+                        [@commitRow.render group commitChecker commit.getCommit() "/courses/${group.course.code}/groups/${group.groupNumber}/pull/${pullRequest.issueId}"]
+                            <span class="pull-right">
+                                <span class="text-success octicon octicon-arrow-up"></span>
+                                <span class="text-muted">${pullRequest.ahead}</span>
+                                <span class="text-danger octicon octicon-arrow-down"></span>
+                                <span class="text-muted">${pullRequest.behind}</span>
+                            </span>
+                            <div class="comment">Pull Request #${pullRequest.issueId}: ${pullRequest.branchName}</div>
+                            <div class="committer">${commit.getMessage()}</div>
+                            <div class="timestamp" data-value="${(commit.getTime() * 1000)?c}">on ${(commit.getTime() * 1000)?number_to_datetime?string["EEEE dd MMMM yyyy HH:mm"]}</div>
+                        [/@commitRow.render]
+                    [/#list]
+                [#else]
+                <tr>
+                    <td class="muted">
+                        No unmerged pull requests in this repository.
+                    </td>
+                </tr>
+                [/#if]
+                </tbody>
+            </table>
+
+            <h4>Closed pull requests</h4>
+            <table class="table table-bordered">
+                <tbody>
+                [#if closedPullRequests?? && closedPullRequests?has_content]
+                    [#list closedPullRequests as pullRequest]
+                        [#assign commit = repository.retrieveCommit(pullRequest.destination)]
+                        [@commitRow.render group commitChecker commit.getCommit() "/courses/${group.course.code}/groups/${group.groupNumber}/pull/${pullRequest.issueId}"]
+                        <span class="pull-right">
+                            [#if pullRequest.merged]
+                                <span class="label label-success"><i class="octicon octicon-git-merge"></i> Merged</span>
+                            [#else]
+                                <span class="label label-danger"><i class="octicon octicon-issue-closed"></i> Closed</span>
+                            [/#if]
+                        </span>
+                        <div class="comment">Pull Request #${pullRequest.issueId}: ${pullRequest.branchName}</div>
+                        <div class="committer">${commit.getMessage()}</div>
+                        <div class="timestamp" data-value="${(commit.getTime() * 1000)?c}">on ${(commit.getTime() * 1000)?number_to_datetime?string["EEEE dd MMMM yyyy HH:mm"]}</div>
+                        [/@commitRow.render]
+                    [/#list]
+                [#else]
+                <tr>
+                    <td class="muted">
+                        No unmerged pull requests in this repository.
```

Comment:
Change to "No closed/merged pull requests in this repository." as you can have unmerged pull requests, but they don't show up in this table.